frontend/src/components/Team.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Card, CardImg, CardBody,
+  CardTitle, CardSubtitle, Container
+} from 'reactstrap';
+import Hannah from '../images/teams-pics/Hannah.jpg';
+import Lillie from '../images/teams-pics/Lillie.jpg';
+import Jose from '../images/teams-pics/Jose.png';
+import Kirtan from '../images/teams-pics/Kirtan.png';
+import '../styles/Team.css';
+const Team = (props) => {
+
+    //Teammate constructor
+    function Teammate(name, year, major, univ, pic) {
+        this.name = name;
+        this.year = year;
+        this.major = major;
+        this.univ = univ;
+        this.pic = pic;
+      }
+    //list of teammates
+    var team = [ new Teammate('Hannah ', 'Junior', 'Computer Science', 'The University of Texas at Dallas', Hannah),
+                 new Teammate('Aru', 'Sophomore', 'Computer Science', 'The University of Texas at Dallas', Lillie), 
```

Comment:
I think there may be a mistake on the image here.

Reply:
I don't have a pic of Aru yet, so I used Lillie as a placeholder to make sure the sizing was okay.